hs628_tools/Test.py

```diff
@@ -0,0 +1,133 @@
+
+import numpy as np
+from sklearn.model_selection import train_test_split
+
+
+
+
+
+
+class data_split:
+    """
+    Create class data_split to predict a perfect train and test data split ratio.
+
+    defines method to accomplish this task.
+
+    returns a split ratio.
+
+    Identifies Missing values as Nan for any missing value, return train and test dataframe with uniform division
+    of NAN between train and test data per the data split ratio.
+
+    """
+    def __init__(self, data):
+        self.data = data
+
+
+    def test_train():
+        """
+        Reads in csv file using np.genfromtxt, counts number of NANs in the dataset
+
+        """
+
+        data = np.genfromtxt(csv_datafile, delimiter=',', skip_header=1)
+        np.isnan(data).sum() #counts the number of "NANs" in dat
+        #print data.shape
+
+
+
+        #set calculation standard for specific datalengths, this is based on the general rule of thumb
+        #ratio increases as the length of the data increases, so to ensure we have  enough data to train and test
+        #for the smaller datasets
+        
+        if len(data) <= 500:
+            train = 60
+            test = 40
```

Comment:
A somewhat minor point (but seems just important enough for me to mention) ```train``` and ```test``` are a bit generic names, they could be referring to the training and test data.  I think it would make the code slightly more clear if they were named ```train_percent``` and ```test_percent``` or perhaps let them be floating numbers and name them ```train_portion``` and ```test_portion```  (if you even keep the test variable, see below)

Reply:
@EHWUSF  I have applied your recommendation and updated the codes. Thank you!